ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -64,13 +80,53 @@ const styles = StyleSheet.create({
 const LABEL_COL_SIZE_NARROW_LABEL = 1;
 const VALUE_COL_SIZE_NARROW_LABEL = 2;
 
-const LABEL_COL_SIZE_WIDE_LABEL = 1;
+const LABEL_COL_SIZE_WIDE_LABEL = 2;
 const VALUE_COL_SIZE_WIDE_LABEL = 1;
 
 /**
  * Details of transaction
  */
-export class TransactionDetailsScreen extends React.Component<Props, never> {
+export class TransactionDetailsScreen extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      isTransactionCompleted: true
+    };
+  }
+
+  /**
+   * It sum the amount to pay and the fee requested to perform the transaction
+   * TO DO: it could be provided by API as presentd on header
+   */
+  private getTotalAmount(transaction: Readonly<WalletTransaction>) {
+    return +(transaction.amount + transaction.transactionCost).toFixed(12);
```

Comment:
wait, why do you need the plus here? can you split this operation in 2/3 steps?

Reply:
it convert the result from string to number again.

Reply:
use `parseInt` when converting a string to number, also split this operation in multiple steps, e.g.:

```ts
const totalAmount = transaction.amount + transaction.transactionCost;
...
```